api/Events/PlantEvents/Client/ClientWantsAllPlants.cs

```diff
@@ -20,9 +21,9 @@ public class ClientWantsAllPlants(PlantService plantService, JwtService jwtServi
 {
     public override async Task Handle(ClientWantsAllPlantsDto dto, IWebSocketConnection socket)
     {
-        var email = jwtService.GetEmailFromJwt(dto.Jwt);
+        var email = jwtService.GetEmailFromJwt(dto.Jwt!);
         var plants = await plantService.GetPlantsForUser(email, dto.PageNumber, dto.PageSize);
-        var serverSendsAllPlantsDto = new ServerSendsAllPlants
+        var serverSendsAllPlantsDto = new ServerSendsPlantsForCollection
```

Comment:
maybe this should be renamed to something more generic, like ServerSendsPlants?

Reply:
renamed the event to ServerSendsPlants